<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportWindow</class>
 <widget class="QMainWindow" name="exportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cascadeur : Export</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="create_sets_group">
      <property name="title">
       <string>Export Sets</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="scene_list"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>32</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="add_hik_button">
           <property name="text">
            <string>Add HIK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="create_data_button">
           <property name="text">
            <string>Create Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_data_button">
           <property name="text">
            <string>Delete Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="selection_tab">
          <attribute name="title">
           <string>Selection Set</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="export_data_group">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Exportable Content</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="dynamic_set">
                  <property name="text">
                   <string>Dynamic Set</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="export_list">
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="extra_button_group">
                <property name="leftMargin">
                 <number>4</number>
                </property>
                <property name="topMargin">
                 <number>4</number>
                </property>
                <property name="rightMargin">
                 <number>4</number>
                </property>
                <property name="bottomMargin">
                 <number>4</number>
                </property>
                <item>
                 <widget class="QPushButton" name="add_extras">
                  <property name="text">
                   <string>Add Selected</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="remove_extras">
                  <property name="text">
                   <string>Remove Selected</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rig_tab">
          <attribute name="title">
           <string>Rig Settings</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QGridLayout" name="qrig_group">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>16</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="chest_joint_label">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Chest Joint</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="clear_left_weapon">
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>X</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="spine_list"/>
             </item>
             <item row="1" column="1">
              <widget class="DropLineEdit" name="left_weapon_node">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>drag-n-drop joint here</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="left_weapon_label">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Left Weapon Joint</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="DropLineEdit" name="right_weapon_node">
               <property name="placeholderText">
                <string>drag-n-drop joint here</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="clear_right_weapon">
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>X</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="right_weapon_label">
               <property name="text">
                <string>Right Weapon Joint</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="checkbox_group">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="align_pelvis">
               <property name="text">
                <string>align pelvis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="create_layers">
               <property name="text">
                <string>create layers</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="twist_settings" native="true">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="4" column="4">
               <widget class="QComboBox" name="right_forearm_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QComboBox" name="left_forearm_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="4">
               <widget class="QComboBox" name="right_arm_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QLabel" name="right_forearm">
                <property name="text">
                 <string>Right Forearm</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="5">
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="left_upper_leg">
                <property name="text">
                 <string>Left Upper Leg</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QComboBox" name="global_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="7" column="4">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="global_twist_axis">
                <property name="text">
                 <string>Global Twist Axis</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>16</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="right_arm">
                <property name="text">
                 <string>Right Arm</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="left_forearm">
                <property name="text">
                 <string>Left Forearm</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="left_arm">
                <property name="text">
                 <string>Left Arm</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QComboBox" name="left_upper_leg_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QComboBox" name="left_leg_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QComboBox" name="left_arm_twist">
                <property name="currentText">
                 <string>X</string>
                </property>
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="6" column="3">
               <widget class="QLabel" name="right_leg">
                <property name="text">
                 <string>Right Leg</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="5" column="3">
               <widget class="QLabel" name="right_upper_leg">
                <property name="text">
                 <string>Right Upper Leg</string>
                </property>
               </widget>
              </item>
              <item row="6" column="4">
               <widget class="QComboBox" name="right_leg_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLabel" name="left_leg">
                <property name="text">
                 <string>Left Leg</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="5" column="4">
               <widget class="QComboBox" name="right_upper_leg_twist">
                <item>
                 <property name="text">
                  <string>X</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="strength_percents">
                <property name="maximumSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>%</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Strength</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="rig_export_group">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Export To</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="rig_current_button">
                <property name="text">
                 <string>Current Scene</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rig_new_button">
                <property name="text">
                 <string>New Scene</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropLineEdit</class>
   <extends>QLineEdit</extends>
   <header>cg3dmaya/cascadeur/editor.py</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
